src/sidebar/components/annotation-omega.js

```diff
@@ -62,9 +66,19 @@ function AnnotationOmega({
     createDraft(annotation, { ...draft, text });
   };
 
+  const onSave = () => {
+    // TODO flash errors if not authenticated or if annotation is empty?
```

Comment:
The current implementation in the `Annotation` controller starts off by flashing errors for a couple of states:

https://github.com/hypothesis/client/blob/ec63961a743dc15e7112290816e19b38ac5d73e5/src/sidebar/components/annotation.js#L265-L272

Both of these states _should never happen_, as in the publish button either doesn't exist or is disabled in these states, making it impossible for the `onSave` callback to be invoked. Unless there's something I'm not thinking of here. Which is why I mention it!

Reply:
I believe you're right Lyza. I think this may be a vestige of an older UI in which the publish button was not disabled in these cases.